inline cgo function breaks mac build

The following cgo functions in native_darwin.go breaks my mac build. The problem is the "inline" modifier. After removing "inline" modifiers, build passes. I am not sure how "inline" is even possible with cgo. Inline function doesn't generate a symbol. As such, cgo compiler won't be able to link with it.

The offending code
```
inline void *workaround_glfwGetCocoaWindow(GLFWwindow *w) {
	return (void *)glfwGetCocoaWindow(w);
}
inline void *workaround_glfwGetNSGLContext(GLFWwindow *w) {
	return (void *)glfwGetNSGLContext(w);
}
```

Build errors

```
~/code/go/src/github.com/go-gl/glfw/v3.1/glfw
14:32 $ go build
# github.com/go-gl/glfw/v3.1/glfw
Undefined symbols for architecture x86_64:
  "_workaround_glfwGetCocoaWindow", referenced from:
      __cgo_d9a1927387a1_Cfunc_workaround_glfwGetCocoaWindow in native_darwin.cgo2.o
     (maybe you meant: __cgo_d9a1927387a1_Cfunc_workaround_glfwGetCocoaWindow)
  "_workaround_glfwGetNSGLContext", referenced from:
      __cgo_d9a1927387a1_Cfunc_workaround_glfwGetNSGLContext in native_darwin.cgo2.o
     (maybe you meant: __cgo_d9a1927387a1_Cfunc_workaround_glfwGetNSGLContext)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

My go env
```
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/John_Song/code/go"
GORACE=""
GOROOT="/usr/local/Cellar/go/1.5/libexec"
GOTOOLDIR="/usr/local/Cellar/go/1.5/libexec/pkg/tool/darwin_amd64"
GO15VENDOREXPERIMENT=""
CC="clang"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fno-common"
CXX="clang++"
CGO_ENABLED="1"
```

--- a/v3.1/glfw/native_darwin.go
+++ b/v3.1/glfw/native_darwin.go
@@ -8,10 +8,10 @@ package glfw
 
 // workaround wrappers needed due to a cgo and/or LLVM bug.
 // See: https://github.com/go-gl/glfw/issues/136
-inline void *workaround_glfwGetCocoaWindow(GLFWwindow *w) {
+void *workaround_glfwGetCocoaWindow(GLFWwindow *w) {
 	return (void *)glfwGetCocoaWindow(w);
 }
-inline void *workaround_glfwGetNSGLContext(GLFWwindow *w) {
+void *workaround_glfwGetNSGLContext(GLFWwindow *w) {
 	return (void *)glfwGetNSGLContext(w);
 }
 */

